crates/compiler/test_syntax/tests/test_fmt.rs

```diff
@@ -2959,6 +2933,142 @@ mod test_fmt {
 
         expr_formats_same(indoc!(
             r"
+            identity = |a,
+                b,
+                # it's c!!
+                c
+                | a
+
+            identity 43
+            "
+        ));
+        // Starting from old syntax
+        expr_formats_to(
+            indoc!(
+                r"
+            identity = \a -> a
+
+            identity 42
+            "
+            ),
+            indoc!(
+                r"
+            identity = |a| a
+
+            identity 42
+            "
+            ),
+        );
+
+        expr_formats_to(
+            indoc!(
+                r"
+            identity = \a ->
+                a
+
+            identity 44
+            "
+            ),
+            indoc!(
+                r"
+            identity = |a|
+                a
+
+            identity 44
+            "
+            ),
+        );
+
+        expr_formats_to(
+            indoc!(
+                r"
+            identity = \a -> a
+
+            # Hello
+            identity 40
+            "
+            ),
+            indoc!(
+                r"
+            identity = |a| a
+
+            # Hello
+            identity 40
+            "
+            ),
+        );
+
+        expr_formats_to(
+            indoc!(
+                r"
+                    identity = \a
+                        -> a
+
+                    identity 41
+                "
+            ),
+            indoc!(
+                r"
+                    identity = |a| a
+
+                    identity 41
+                "
+            ),
+        );
+
+        expr_formats_to(
+            indoc!(
+                r"
+                    identity = \a
+                        ->
+                            a + b
+
+                    identity 4010
+                "
+            ),
+            indoc!(
+                r"
+                    identity = |a|
+                        a + b
+
+                    identity 4010
+                "
+            ),
+        );
+
+        expr_formats_to(
+            indoc!(
+                r"
+            identity = \a, b -> a
+
+            identity 43
+            "
+            ),
+            indoc!(
+                r"
+            identity = |a, b| a
+
+            identity 43
+            "
+            ),
+        );
+
+        // expr_formats_same(indoc!(
```

Comment:
should this be removed?

Reply:
Replaced this with some real testing of collapsing a small record arg